app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsTable/index.tsx

```diff
@@ -0,0 +1,185 @@
+    } = useRequest({
+        skip: isNotDefined(countryResponse?.iso3),
+        url: '/api/v2/local-units/',
+        preserveResponse: true,
+        query: {
+            limit,
+            offset,
+            type__code: filter?.type,
+            validated: isDefined(filter?.isValidated)
+                ? filter.isValidated === VALIDATED : undefined,
+            search: filter?.search,
+            country__iso3: isDefined(countryResponse?.iso3) ? countryResponse?.iso3 : undefined,
+        },
+    });
+
+    const columns = useMemo(
+        () => ([
+            createStringColumn<LocalUnitsTableListItem, number>(
+                'branch_name',
+                strings.localUnitsTableName,
+                (item) => item.local_branch_name ?? item.english_branch_name,
+            ),
+            createStringColumn<LocalUnitsTableListItem, number>(
+                'address',
+                strings.localUnitsTableAddress,
+                (item) => item.address_loc ?? item.address_en,
+            ),
+            createStringColumn<LocalUnitsTableListItem, number>(
+                'type',
+                strings.localUnitsTableType,
+                (item) => item.type_details.name,
+            ),
+            createStringColumn<LocalUnitsTableListItem, number>(
+                'focal',
+                strings.localUnitsTableFocal,
+                (item) => item.focal_person_en,
+            ),
+            createStringColumn<LocalUnitsTableListItem, number>(
+                'phone',
+                strings.localUnitsTablePhoneNumber,
+                (item) => item.phone,
+            ),
+            createStringColumn<LocalUnitsTableListItem, number>(
+                'email',
+                strings.localUnitsTableEmail,
+                (item) => item.email,
+            ),
+            createBooleanColumn<LocalUnitsTableListItem, number>(
+                'validated',
+                strings.localUnitsTableValidated,
+                (item) => item.validated,
+            ),
+            createElementColumn<LocalUnitsTableListItem, number, LocalUnitsTableActionsProps>(
+                'actions',
+                '',
+                LocalUnitsTableActions,
+                (_, item) => ({
+                    localUnitId: item.id,
+                    isValidated: item.validated,
+                    localUnitName: item.local_branch_name ?? item.english_branch_name,
+                    onActionSuccess: refetchLocalUnits,
+                }),
+            ),
+        ]),
+        [
+            strings.localUnitsTableAddress,
+            strings.localUnitsTableName,
+            strings.localUnitsTableType,
+            strings.localUnitsTableFocal,
+            strings.localUnitsTablePhoneNumber,
+            strings.localUnitsTableEmail,
+            strings.localUnitsTableValidated,
+            refetchLocalUnits,
+        ],
+    );
+
+    return (
+        <Container
+            footerContent={(
```

Comment:
Check when locationUnitesTableResponse is not defined

Reply:
Table component will handle the undefined